### DISCUSSION TITLE
Windows Live Mail Not Working Properly Under Sandboxie Classic 5.52

### DISCUSSION THREAD

**Discussion by w7rbl** [sandboxie-plus/Sandboxie#1267] [Q&A Feedback] [answered]
Windows Live Mail (with Windows 7 64Bit (Pro & Home)) does not work properly with the new versions of Sandboxie Classic 5.52 (all versions) - the problem is that emails keep coming in even when they are deleted - i.e incoming emails continue to be duplicated time after time.

**Comment by isaak654** [comment, marked as answer]
Reposted in issue [#1272](https://github.com/sandboxie-plus/Sandboxie/issues/1272).

**Comment by DavidXanatos** [comment]
Does it worked properly with any previous version? If so please test which exact version introduced this issue.

**Reply by isaak654** [reply]
This thread is locked, but he can reply on issue #1272.
